tests/unittests/BackendTest.cpp

```diff
@@ -499,3 +499,13 @@ INSTANTIATE_TEST_CASE_P(JIT, BackendTest, ::testing::Values(BackendKind::CPU));
 INSTANTIATE_TEST_CASE_P(OpenCL, BackendTest,
                         ::testing::Values(BackendKind::OpenCL));
 #endif // GLOW_WITH_OPENCL
+
+/// Check if the dump function works for Type.
+TEST(BackendTest, dumpType) {
+  Module mod;
+  TypeRef tyA = mod.uniqueType(ElemKind::FloatTy, {1, 32, 32, 3});
+  tyA->dump();
```

Comment:
It is probably better to create a custom stream object and output into this stream, so that we do not pollute the real estate of the terminal during testing. You could even take this output stream and then compare it with the result of `toString`.

Reply:
> It is probably better to create a custom stream object and output into this stream, so that we do not pollute the real estate of the terminal during testing. You could even take this output stream and then compare it with the result of `toString`.

OK, Thanks. I created a separate `llvm::raw_string_ostream` to verify the results. So the terminal looks clean.